### PR TITLE
 added httpMethod in perform action parameter and default to http.Get

### PR DESCRIPTION
**Pull request by xavave**

### CODE REVIEW

**Review by AmenJlili — APPROVED** [review]
Should be good... nothing to see here

File: src/LicenseManager.cs [hunk 1]
```diff
@@ -222,30 +224,31 @@ public ILicenseResponse PerformAction(Consts.Action action, License license)
...
+
         }
-    
-        public Task<ILicenseResponse> PerformActionAsync(Consts.Action action, License License) 
+
+        public Task<ILicenseResponse> PerformActionAsync(Consts.Action action, License License, RestSharp.Method httpMethod = RestSharp.Method.GET)
         {
-            return Task.Run(() => {
+            return Task.Run(() =>
+            {
```
Comment (AmenJlili) [inline src/LicenseManager.cs:265]: @xavave  I think you need to inject httpMethod synchronous method

File: Tests/WPLMNETClient.NetFramework.Tests/Tests.cs [hunk 2]
```diff
@@ -18,19 +18,19 @@ public void OnStartUp()
         {
```
Comment (AmenJlili) [inline Tests/WPLMNETClient.NetFramework.Tests/Tests.cs:19]: We should prolly add a little  comment to the user the configuration and license object are specific to their website

**Review by AmenJlili — CHANGES_REQUESTED** [review]

**Comment by xavave** [comment]
I'm not sure to fully understand your change request: anyway, I've modified configuration parameters (which were specific to your debugging server) 
I've also I've found and fixed another  issue:  the 3 dates sent to server (date_created,date_renewed,date_expiry) were sent empty, because server is waiting for this date type: yyyy-MM-dd so I've parsed these datetimes with ToString("yyyy-MM-dd") and, now the dates are properly sent to the server.

the first issue was not related to synchronous or asynchronous method (I was already using synchronous method), but due to the fact that my client App (a WPF app) is not located on the wordpress server (which is on my synology NAS) so I guess that when you want to POST method from an external location to the wordpress server, wordpress POST client request needs to be authentified , whereas with GET method , no need to authenticate with a wordpress account /token before sending the request

**Comment by AmenJlili** [comment]
I'm talking about this : 
image
You forget to pass the optional parameter... 

I'll fix that myself no problem. Thanks for your commits! Merci beaucoup!

**Pull request merged**